tests/.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    embertest: true
+  },
+  globals: {
+    capture: true,
+    chai: true,
```

Comment:
Ok I will leave it up to you for this one. Maybe we can do the change or add an issue on the repo.